### PR TITLE
[Prices] Add 387 Missing Token Prices

### PR DESCRIPTION
**Pull request by bh2smith**
Similar to yesterday's addition of 50 tokens (#1639), today we attempt to add another 387.

This was built with a script that fetches all active [tokens from coin paprika](https://api.coinpaprika.com/v1/coins) and "joining" on the results from their [contracts endpoint](https://api.coinpaprika.com/v1/contracts/eth-ethereum). 

We then take the results of this [tokens with missing prices query](https://dune.com/queries/1317238) and match whatever we can (uniquely by address).

In the future we can clean up the script and generalize to make this work for multiple networks (at least those which [coin paprika supports](https://api.coinpaprika.com/v1/contracts/))

cc @soispoke

### CODE REVIEW

File: models/prices/prices_tokens.sql [hunk 1]
```diff
@@ -600,7 +600,6 @@ VALUES
     ("hop-hop-protocol", "ethereum", "HOP", "0xc5102fe9359fd9a28f877a67e36b0f050d81a3cc", 18),
     ("x2y2-x2y2token", "ethereum", "X2Y2", "0x1e4ede388cbc9f4b5c79681b7f94d36a11abebc9", 18),
     ("tcr-tracer-dao", "ethereum", "TCR", "0x9c4a4204b79dd291d6b6571c5be8bbcd0622f050", 18),
-    ("wsteth-wrapped-liquid-staked-ether-20", "ethereum", "WSTETH", "0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0", 18),
```
Comment (bh2smith) [inline models/prices/prices_tokens.sql:603]: This was duplicated see main:

https://github.com/duneanalytics/spellbook/blob/0c5b439f2006c96f045c4c50feabcb82a3629209/models/prices/prices_tokens.sql#L588-L604


Added in both #1628 and #1639

**Comment by soispoke** [comment]
@aalan3 Did everything go well when we added 50 tokens yesterday ? Should we go ahead with this ?

**Comment by bh2smith** [comment]
> @aalan3 Did everything go well when we added 50 tokens yesterday ? Should we go ahead with this ?

One thing I noticed is that wstETH got duplicated. We may want to add a test for duplicated tokens on this table. 

[This query](https://dune.com/queries/1319977?d=1) shows that wstETH has not yet been added (possibly due to the duplication). 

However it looks like others have. Example:

- [STRNGR](https://dune.com/queries/1319977?d=1&TokenContract_t6c1ea=0xdc0327d50e6c73db2f8117760592c8bbf1cdcf38)

**Review by aalan3 — APPROVED** [review]
This is quite a lot but let's try it and see what happens.

**Comment by soispoke** [comment]
lgtm then 🚀, merge and deployed let's see how it goes 🙏 
@aalan3 tagging you for visibility

**Comment by bh2smith** [comment]
Everything is looking good so far here except for the one manually entered token XMON that was included. 

If we run the PoC query again we see that all the added tokens are no longer returned as "missing" except for XMON.

**Pull request merged**